<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QmitkIGTLDeviceSourceManagementWidgetControls</class>
 <widget class="QWidget" name="QmitkIGTLDeviceSourceManagementWidgetControls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>443</width>
    <height>781</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="label">
       <property name="toolTip">
        <string>The selected IGTL device source</string>
       </property>
       <property name="text">
        <string>Selected IGTL Device Source: </string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignRight">
      <widget class="QLabel" name="selectedSourceLabel">
       <property name="text">
        <string>&lt;none&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="setupConnectionGroupBox">
     <property name="title">
      <string>Setup Connection</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QmitkIGTLDeviceSetupConnectionWidget" name="connectionSetupWidget" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="sendStringsGroupBox">
     <property name="title">
      <string>Send String Messages</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLineEdit" name="editSend">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>Enter the string to be sent</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="butSend">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>Sends a message with the given string to the connected device</string>
        </property>
        <property name="text">
         <string>Send String</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="sendCommandsGroupBox">
     <property name="title">
      <string>Send Command Messages</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QmitkIGTLDeviceCommandWidget" name="commandWidget" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QmitkIGTLDeviceCommandWidget</class>
   <extends>QWidget</extends>
   <header>QmitkIGTLDeviceCommandWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QmitkIGTLDeviceSetupConnectionWidget</class>
   <extends>QWidget</extends>
   <header>QmitkIGTLDeviceSetupConnectionWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
